spec/index.bs

```diff
@@ -12387,29 +12387,36 @@ attachments used by this encoder.
                 1. For each |bundle| in |bundles|:
                     1. Increment |this|.{{GPURenderCommandsMixin/[[drawCount]]}} by |bundle|.{{GPURenderBundle/[[drawCount]]}}.
 
-                1. [=map/Clear=] |this|.{{GPUBindingCommandsMixin/[[bind_groups]]}}.
-                1. Set |this|.{{GPURenderCommandsMixin/[[pipeline]]}} to `null`.
-                1. Set |this|.{{GPURenderCommandsMixin/[[index_buffer]]}} to `null`.
-                1. [=map/Clear=] |this|.{{GPURenderCommandsMixin/[[vertex_buffers]]}}.
+                    1. [$Reset the render pass bindings$] of |this|.
+                    1. Let |passState| be a snapshot of |this|'s current state.
+                    1. [$Enqueue a render command$] on |this| which issues the following steps on the
+                        [=Queue timeline=] with |renderState| when executed:
 
-                1. Let |passState| be a snapshot of |this|'s current state.
-                1. [$Enqueue a render command$] on |this| which issues the subsequent steps on the
-                    [=Queue timeline=] with |renderState| when executed.
-            </div>
-            <div data-timeline=queue>
-                [=Queue timeline=] steps:
+                        <div data-timeline=queue>
+                            [=Queue timeline=] steps:
 
-                1. For each |bundle| in |bundles|:
-                    1. Execute each command in |bundle|.{{GPURenderBundle/[[command_list]]}}
-                        with |passState| and |renderState|.
```

Comment:
... I'm pretty sure snapshotting `passState` and using it here was wrong. [Render commands](https://gpuweb.github.io/gpuweb/#abstract-opdef-enqueue-a-render-command) don't take `passState` as an input; they've already captured their bundle's local `passState` when they were encoded. (`passState` should be renamed, since both bundles and passes have it. The counterpart of `RenderState` is the combination of `GPURenderCommandsMixin` and `GPUBindingCommandsMixin`, which is not helpful.... I think `encoderState` would be good.)

Reply:
Done. As discussed offline used `bindingState` instead. Thanks for the feedback!

Reply:
offline discussion: use bindingState instead of encoderState because we already have "validate the encoder state" which is a different thing.